src/genie/libs/parser/ios/tests/ShowInterfacesSwitchport/cli/equal/golden_output_2_expected.py

```diff
@@ -31,7 +31,7 @@
         "switchport_enable": True,
         "private_vlan": {},
         "capture_mode": False,
-        "trunk_vlans": "1,111,130,131,400,405,410,420,430,439-442,450,451,460,",
+        "trunk_vlans": "1,111,130,131,400,405,410,420,430,439-442,450,451,460,470,480,490,500,616,619,700,709-712,720,723-725,760",
```

Comment:
Please run unit test and attach screen shot.

Reply:
![image](https://user-images.githubusercontent.com/19796896/111236327-d65e0d80-85f2-11eb-9193-5bda43e6faa3.png)